catroid/src/main/java/org/catrobat/catroid/ui/fragment/LookFragment.java

```diff
@@ -267,6 +268,8 @@ public void setOnLookDataListChangedAfterNewListener(OnLookDataListChangedAfterN
 
 	@Override
 	public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+		SnackbarUtil.showSnackbar(getActivity(), getActivity().getResources().getString(R.string.hint_looks), getActivity().getResources().getResourceName(R.string.hint_looks));
```

Comment:
[`getString(...)`](https://github.com/android/platform_frameworks_base/blob/0acc91e4fcb00d942335f72b500d4812d141ea8d/core/java/android/content/Context.java#L474-L477) is a shorthand for `getResources().getString(...)`. Update all remaining occurrences.